src/components/popups/TeamSetting.tsx

```diff
@@ -147,11 +87,13 @@ const TeamSettings = ({ isOpen, onClose, onEdit, setIsLeave }) => {
         },
       });
       onEdit();
+      setIsLoading(false);
       DeactivateEditMode();
     } catch (error) {
       setError("Something went wrong. Please try again");
       console.error("Error while updating Team", handleError(error));
     }
+    setIsLoading(false);
```

Comment:
`setError` false?

Reply:
Need further explanation what is asked for?